packages/commonwealth/client/scripts/views/pages/view_proposal/index.tsx

```diff
@@ -41,30 +41,18 @@ import {
 } from 'state/api/proposals';
 import { usePoolParamsQuery } from 'state/api/chainParams';
 
-type ViewProposalPageAttrs = {
-  identifier: string;
-  type?: string;
-};
-
-const ViewProposalPage = ({
-  identifier,
-  type: typeProp,
-}: ViewProposalPageAttrs) => {
-  const proposalId = identifier.split('-')[0];
-  const navigate = useCommonNavigate();
+export function useProposalData(proposalId, typeProp, ignoreInitChain = false) {
```

Comment:
similar comment as this one https://github.com/hicommonwealth/commonwealth/pull/5115#discussion_r1337446465